lib/futurism/channel.rb

```diff
@@ -37,11 +38,17 @@ def receive(data)
     def resource(signed_params:, sgid:)
       return GlobalID::Locator.locate_signed(sgid) if sgid.present?
 
-      Rails
-        .application
-        .message_verifier("futurism")
+      message_verifier
         .verify(signed_params)
         .deep_transform_values { |value| value.is_a?(String) && value.start_with?("gid://") ? GlobalID::Locator.locate(value) : value }
     end
+
+    def controller(signed_controller:)
+      return ApplicationController unless signed_controller.present?
+
+      message_verifier
+        .verify(signed_controller)
+        .yield_self { |controller_string| Kernel.const_get(controller_string) }
+    end
```

Comment:
I believe we want to sign the controller class because we don't want it to be tampered with client-side

Reply:
What’s the advantage of `Kernel.const_get` over `constantize`?

Reply:
very little. I'll make the change

Reply:
Thanks, I was just thinking, we are in a Rails context anyway... let's keep the syntactic sugar 😁